core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -576,6 +590,13 @@ case class GlobalPSBTMap(elements: Vector[GlobalPSBTRecord]) extends PSBTMap {
       .filter(element => PSBTGlobalKeyId.fromByte(element.key.head) == key)
       .asInstanceOf[Vector[T]]
   }
+
+  def combine(other: GlobalPSBTMap): GlobalPSBTMap = {
```

Comment:
scaladocs for all of these and I would prefer more descriptive names to overloading

Reply:
Wasn't really sure what a more descriptive name could be, but I think the added scaladocs should better explain